README.md

```diff
@@ -52,6 +52,26 @@ $ ruby test/payment_token_test.rb
 5 tests, 18 assertions, 0 failures, 0 errors, 0 skips
 ```
 
+## Generating the Apple Root Certificate
+
+In order to get the certificate into a usable format, the following java code was used.
+```
+  InputStream inputStream = null;
+  X509Certificate appleRootCertificate = null;
+
+  try {
+    InputStream in = new FileInputStream(new File("/Users/mrezentes/Documents/workspace/AndroidPay/src/AppleRootCA-G3.cer"));
+    CertificateFactory certificateFactory = CertificateFactory.getInstance(X_509);
+    appleRootCertificate = (X509Certificate) certificateFactory.generateCertificate(in);
+    PEMWriter pw = new PEMWriter(new PrintWriter(System.out));
+    pw.writeObject(appleRootCertificate);
+    pw.flush();
+    pw.close();
+  }
+
+```
+
```

Comment:
This is an attempt to help future Miki or some other future dev.  I will make a notes.md and put it there.